src/fmdb/FMDatabase.m

```diff
@@ -454,7 +454,7 @@ - (BOOL)databaseExists {
     
     if (!_db) {
             
-        NSLog(@"The FMDatabase %@ is not open.", self);
+        [self log:SFLogLevelDebug format:@"The FMDatabase %@ is not open.", self];
```

Comment:
This looks like an error case: the app is going to crash if configured to do so.